### PR TITLE
DCT-323D

### PR DESCRIPTION
**Pull request by Craig-Elre**
Create the remaining folders for the Wisdom For Sale tutorial

### CODE REVIEW

File: examples/docs-wisdom-yargs/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM reachsh/runner:0.1.10
+
+# If your project needs more Node dependencies, uncomment these next three
+# lines by removing the leading `#` character:
+
+ COPY package.json /app/package.json
+ RUN npm install
+ RUN npm link @reach-sh/stdlib
```
Comment (hamirmahal) [inline examples/docs-wisdom-yargs/Dockerfile:8]: I think I might be missing something. These three lines are already uncommented.

Comment (Craig-Elre) [reply examples/docs-wisdom-yargs/Dockerfile:8]: I need to remove this file and a few others.

File: examples/docs-wisdom-11/index.mjs [hunk 2]
```diff
@@ -32,7 +32,7 @@ if (role === 'seller') {
     }
   };
 		
-  const acc = await stdlib.newTestAccount(stdlib.parseCurrency(iBalance));
+  const acc = await stdlib.newTestAccount(iBalance);
   await showBalance(acc);
   const ctc = acc.contract(backend);
   await ctc.participants.Seller(sellerInteract)
```
Comment (TheChronicMonster) [inline examples/docs-wisdom-11/index.mjs:38]: Missing semicolon

File: examples/docs-wisdom-12/index.mjs [hunk 3]
```diff
@@ -39,7 +39,7 @@ if (role === 'seller') {
     }
   };
 		
-  const acc = await stdlib.newTestAccount(stdlib.parseCurrency(iBalance));
+  const acc = await stdlib.newTestAccount(iBalance);
   await showBalance(acc);
   const ctc = acc.contract(backend);
   await ctc.participants.Seller(sellerInteract)
```
Comment (TheChronicMonster) [inline examples/docs-wisdom-12/index.mjs:45]: Missing semicolon

Comment (TheChronicMonster) [inline examples/docs-wisdom-12/index.mjs:45]: Missing semicolon

**Review by TheChronicMonster — APPROVED** [review]
Tested directories 7-14 to ensure `reach compile`, `reach run index seller` and `reach run index buyer` run without errors.

**Review by jeapostrophe — APPROVED** [review]

**Pull request merged**